### PR TITLE
Move aws job use e2e scenario

### PR DESCRIPTION
**Pull request by krzyzacy**

### CODE REVIEW

File: jenkins/bootstrap_test.py [hunk 1]
```diff
@@ -1446,34 +1446,18 @@ def Check(job, name):
...
-        ]
-
-        is_modern = lambda name: any(re.match(w, name) for w in whitelist) or not any(re.match(b, name) for b in blacklist)
         def Check(job, name):
             job_name = 'ci-%s' % name
             self.assertIn('frequency', job)
             self.assertIn('trigger-job', job)
             self.assertNotIn('branch', job)
-            if is_modern(name):
-                self.assertNotIn('json', job)
```
Comment (ixdy) [inline jenkins/bootstrap_test.py]: maybe keep this?

File: scenarios/kubernetes_e2e.py [hunk 2]
```diff
@@ -84,6 +84,12 @@ def add_files(self, env_files):
                         continue
                     self.env_files.append(self.parse_env(line))
 
+    def add_aws_cred(self, priv, pub, cred):
+        """Sets aws keys and credentials."""
+        self.add_environment('JENKINS_AWS_SSH_PRIVATE_KEY_FILE=%s' % priv)
+        self.add_environment('JENKINS_AWS_SSH_PUBLIC_KEY_FILE=%s' % pub)
+        self.add_environment('JENKINS_AWS_CREDENTIALS_FILE=%s' % path)
```
Comment (krzyzacy) [inline scenarios/kubernetes_e2e.py]: do I want to copy the keys to ./~ssh in local mode?

I don't have a key so I don't know :imp:

File: jenkins/bootstrap_test.py (hunk 1 above)
Comment (krzyzacy) [inline jenkins/bootstrap_test.py]: done, will remove when we get rid of the --json flag

**Comment by zmerlynn** [comment]
Rebase and feel free to self-LGTM, assuming it's trivial. Thanks!

**Comment by krzyzacy** [comment]
yeah should only affect aws job, will keep an eye on it.

**Comment by krzyzacy** [comment]
merging on green

**Comment by zmerlynn** [comment]
Verified migration worked. :+1:

**Pull request merged**